mcstatus/tests/test_pinger.py

```diff
@@ -80,43 +83,28 @@ def test_test_ping_wrong_token(self):
 
 
 class TestPingResponse:
```

Comment:
Since the response classes are no longer in `pinger.py` file, it doesn't make sense to test them in `test_pinger.py`, instead, they should have their own new test file.

Reply:
Moved in cc4a8ef5f06d280172c33c07fb5d2c4f185ec3da.